### PR TITLE
Add index.hu family

### PR DESCRIPTION
**Pull request by tacsipacsi**
Add Index.hu and its family (velvet.hu, divany.hu, totalcar.hu, and
travelo.hu) as a new translator file.

### CODE REVIEW

File: Index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+				break;
+			case 'og:title':
+				newItem.title = content;
+				break;
+			case 'og:description':
+				newItem.abstract = content;
+				break;
+			case 'i:publication':
+				newItem.date = content;
+			case 'author':
```
Comment (zuphilip) [inline Index.js]: It looks like you are missing a `break` here.

**Comment by zuphilip** [comment]
Can you give some examples for your translator?

What is the metadata translator at the moment give for them?

**Comment by tacsipacsi** [comment]
Unfortunately I don't know how to test it. I copied the file into the "zotero/translators" folder in my Firefox profile folder, but it doesn't seem to work. I find only "Embedded Metadata" and the two "Save as web page" options. The Embedded Metadata gives:
``` json
[
	{
		"id": "http://zotero.org/users/local/ujhRic8i/items/KCT4E4FC",
		"type": "webpage",
		"title": "Idomeni: borzasztóak voltak a higiéniás körülmények",
		"abstract": "A szimbólummá vált görög menekülttáborban feljött a talajvíz, mindent sár borított. Egy önkéntesként ott dolgozó magyar orvos mesélt az Indexnek a tapasztalatairól, és arról, hogy a menekültek között jó a beoltottság aránya.",
		"URL": "http://index.hu/kulfold/2016/04/10/schuler_zsofia_idomeni_magyar_voroskereszt/",
		"shortTitle": "Idomeni",
		"author": [
			{
				"literal": "Szabó Dániel"
			}
		],
		"accessed": {
			"date-parts": [
				[
					"2016",
					4,
					10
				]
			]
		}
	}
]
```
The custom JS should include some additional static data (type, language, publisher name). My main problem is that the MediaWiki extension [citoid](https://www.mediawiki.org/wiki/Citoid) doesn't use the metadata to produce the citation.

**Comment by zuphilip** [comment]
Do you use scaffold for coding the translators? Testing is easiest and reliably done within scaffold. Moreover, you can code translators nicely in scaffold and immediately run the new code against a specific side.

At the moment, I am mostly interested in a bunch of urls which you would like to use with your translator/citoid.

It is possible to call the metadata translator within your new translator and then just add the missing pieces. Or you write a new independent translator if there is not much useful information in the metadata. I haven't made my mind, what is best for your cases here...

**Comment by tacsipacsi** [comment]
No, I didn't use any specific programs, I coded in Notepad++ and tested a bit in Firefox web console. Where can I find this "scaffold"?

The sites are in the commit message and (as a regexp) in the array at the top of the file. Example batch pages are index pages (http://index.hu/, http://velvet.hu/, http://divany.hu/, http://totalcar.hu/ and http://travelo.hu/), tags (e.g. http://index.hu/24ora/?cimke=panama-iratok), and searches (e.g. http://index.hu/24ora/?s=CAF). Individual articles are e.g. http://index.hu/kultur/eletmod/2016/04/10/petrus_laci_konyha_etterem_budapest_bib_gourmand_michelin/ or http://totalbike.hu/hirek/2016/04/09/ketmillio_forintnyi_terepmotort_loptak_el/.

**Comment by zuphilip** [comment]
Scaffold can be found here https://www.zotero.org/support/dev/translators/scaffold (but the [fix for a recent bug in Firefox 45](https://github.com/zotero/scaffold/compare/v3.2.2...master) is not yet in the release). The links are helpful, thank you. I will be busy the next days, but maybe @aurimasv or @adam3smith want to look closer here.

**Comment by tacsipacsi** [comment]
Thank you, it's a fantastic add-on! Unfortunately my code was full of errors. Now it's better, but when I tried to test it (doWeb), my browser completely froze, so I'll try a bit more, probably with even less open tabs. I don't know when will I have time, but certainly not tonight.

**Comment by zuphilip** [comment]
There is a new version 3.2.3 of Scaffold out since some hours. Maybe you have to update your version. Let us know when you are ready. You could also use the metadata translator within your translator and then just add the missing information. That should work pretty well for your examples, but you would need to rewrite some parts. As an example you can look at this translator: https://github.com/zotero/translators/blob/master/medes.js

**Comment by tacsipacsi** [comment]
It seems that I'm not interested in it anymore. Anyway, it's maybe easier to write it from scratch that fixing my one-and-a-half-year-old-code if I ever get interested again.

**Pull request closed without merging**